src/main/java/duke/Event.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+/**
+ * Task with a period of time of the task
+ */
+public class Event extends Task{
+    /**
+     * Event object at field which indicates the timings
+     */
+    protected String at;
```

Comment:
I'm curious as to whether the access modifier can be private or what goes into the decision to make it protected